docs/testing-tool.rst

```diff
@@ -7,85 +7,118 @@ Use the KSQL testing tool to test a set of KSQL statements. The KSQL testing too
 is a command line utility that enables testing KSQL statements without requiring any infrastructure, like |ak-tm| and KSQL clusters.
 The KSQL testing tool is a great way to design your KSQL pipeline and ensure the expected results are generated.
 You can collaborate on designing your KSQL statements by sharing the test files.
-You provide a JSON file that describes a set of KSQL statements, along with the input data and expected output data.
-Run the testing tool from a terminal and pass the test file as a parameter.
+To test a set of KSQL statements, you provide three files, one file containing the KSQL statements and two JSON files containing the input records and the expected output records.
 
 .. code:: bash
 
-    ksql-testing-tool /path/to/the/test/file.json
+    NAME
+            ksql-test-runner - The KSQL testing tool
+
+    SYNOPSIS
+            ksql-test-runner {--input-file | -i} <inputFile>
+                    {--output-file | -o} <outputFile>
+                    {--sql-file | -s} <statementsFile>
+
+    OPTIONS
+            --input-file <inputFile>, -i <inputFile>
+                A JSON file containing the input records.
+
+                This option may occur a maximum of 1 time
+
+
+            --output-file <outputFile>, -o <outputFile>
+                A JSON file containing the expected output records.
+
+                This option may occur a maximum of 1 time
+
+
+            --sql-file <statementsFile>, -s <statementsFile>
+                A SQL file containing KSQL statements to be tested.
+
+                This option may occur a maximum of 1 time
+
 
 
 Test File Structure
 *******************
-The test file is a JSON file containing the KSQL statements, input data, desired configurations, and the expected results or expected errors.
-The following is a sample test file:
+
+Statements File
+---------------
+
+The statements file contains the KSQL statements to test. The following are the supported statements in the testing tool:
+
+- CREATE STREAM
+- CREATE TABLE
+- CREATE STREAM AS SELECT
+- CREATE TABLE AS SELECT
+- INSERT INTO SELECT
+
+Here is a sample statements file for the testing tool:
+
+.. code:: sql
+    CREATE STREAM SRTREAM1 (ID bigint, NAME varchar, VALUE double) WITH (kafka_topic='test_topic', value_format='DELIMITED', key='ID');
```

Comment:
```suggestion
    CREATE STREAM STREAM1 (ID bigint, NAME varchar, VALUE double) WITH (kafka_topic='test_topic', value_format='DELIMITED', key='ID');
```
nit: spelling. Same comment for the line below. (Updating this also requires updating the example inputs and outputs.)

Reply:
Ah, how did I miss this?!!!! Fixed it :)